apps/meteor/app/emoji/client/helpers.ts

```diff
@@ -156,6 +156,15 @@ export const updateRecent = (recentList: string[]) => {
 	});
 };
 
+export const replaceEmojiInRecent = (oldEmoji: string, newEmoji: string) => {
```

Comment:
I think using an object here would be better, so the caller can know whats needs to be provided. WDYT?